test/rate_test.py

```diff
@@ -8,23 +8,30 @@
 requests with a predefined sequence of responses.
 """
 
-import unittest
+import doctest
+import json
 import socket
 import time
-import doctest
+import unittest
 
 import json
 import pydocumentdb.https_connection
 import pydocumentdb.documents as documents
 from collections import deque
 
+import pydocumentdb.https_connection
+from pydocumentdb import document_client
+
+MASTER_KEY = ''
```

Comment:
There is a masterKey defined inside the RateTest class. You can move this definition of MASTER_KEY inside the class and change the case for HOST as well and use both of them as a class specific variables.